src/Management/src/Endpoint/Actuators/HttpExchanges/HttpExchangesEndpointOptions.cs

```diff
@@ -0,0 +1,81 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+using Steeltoe.Management.Configuration;
+
+namespace Steeltoe.Management.Endpoint.Actuators.HttpExchanges;
+
+public sealed class HttpExchangesEndpointOptions : EndpointOptions
+{
+    /// <summary>
+    /// Gets or sets a value indicating how many traces should be stored. Default value is 100.
+    /// </summary>
+    public int Capacity { get; set; } = -1;
+
+    /// <summary>
+    /// Gets or sets a value indicating whether HTTP headers from the request should be included in traces.
+    /// <para>
+    /// If a request header is not present in the <see cref="RequestHeaders" />, the header name will be logged with a redacted value. Request headers can
+    /// contain authentication tokens, or private information which may have regulatory concerns under GDPR and other laws. Arbitrary request headers should
+    /// not be logged unless logs are secure and access controlled and the privacy impact assessed.
+    /// </para>
+    /// </summary>
+    public bool AddRequestHeaders { get; set; } = true;
+
+    /// <summary>
+    /// Gets or sets a value indicating whether HTTP headers from the response should be included in traces.
+    /// <para>
+    /// If a response header is not present in the <see cref="ResponseHeaders" />, the header name will be logged with a redacted value.
+    /// </para>
+    /// </summary>
+    public bool AddResponseHeaders { get; set; } = true;
+
+    /// <summary>
+    /// Gets or sets a value indicating whether the request path should be included in traces.
+    /// </summary>
+    public bool AddPathInfo { get; set; } = true;
```

Comment:
Based on the descriptions, names such as `IncludePathInfo` seem more appropriate. Unless this is for spring-compatibility.